config/llama2_7b.yaml

```diff
@@ -0,0 +1,22 @@
+data: !include data/openwebtext_source.yaml
```

Comment:
can't use this b/c the tokenizer is different

Reply:
Yes, I will switch to a new dataset once I finished tokenization and caching.